packages/new-polymath-issuer/src/components/enhancers/index.tsx

```diff
@@ -1,2 +1,3 @@
 export { DataFetcher } from './DataFetcher';
 export { PageLoader } from './isPage';
+export { ParseCsv, RenderProps } from './ParseCsv';
```

Comment:
I think you don't need to export the `RenderProps` type. The enhancer should provide these out of the box